tests/difference/core/driver/core_test.go

```diff
@@ -237,7 +237,6 @@ func (s *CoreSuite) matchState() {
 		for j := 0; j < initState.NumValidators; j++ {
 			s.Require().Equalf(int64(s.traces.Tokens(j)), s.providerTokens(int64(j)), diagnostic+"P tokens mismatch for val %d", j)
 		}
-		// TODO: delegations
```

Comment:
Not a good idea/ not worth doing. The method is blackbox and delegations are implementation detail.
More docs/ onboarding material is coming which will explain method and reasoning.